### PR TITLE
Initial code review

### PR DESCRIPTION
**Pull request by C-D-Lewis**

### CODE REVIEW

File: pebble/src/windows/main_window.c [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+static void click_config_provider(void *context) {
+  window_single_click_subscribe(BUTTON_ID_SELECT, select_click_handler);
+}
+
+static void window_load(Window *window) {
+  Layer *window_layer = window_get_root_layer(window);
+  GRect bounds = layer_get_bounds(window_layer);
+
+  s_text_layer = text_layer_create(GRect(bounds.origin.x, bounds.origin.y + 50, bounds.size.w, bounds.size.h));
```
Comment (orviwan) [inline pebble/src/windows/main_window.c:81]: should: bounds.size.h - 50

File: AccelDataStreamAndroid/app/src/main/java/com/pebble/acceldatastreamandroid/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+import com.getpebble.android.kit.PebbleKit;
+import com.getpebble.android.kit.util.PebbleDictionary;
+
+import java.util.UUID;
+
+public class MainActivity extends AppCompatActivity {
+
+    private static final String TAG = MainActivity.class.getName();
+    private static final UUID APP_UUID = UUID.fromString("bb039a8e-f72f-43fc-85dc-fd2516c7f328");
```
Comment (orviwan) [inline AccelDataStreamAndroid/app/src/main/java/com/pebble/acceldatastreamandroid/MainActivity.java:20]: UUID must match watchapp

**Comment by orviwan** [comment]
screenshots, readme and licence?

File: pebble/src/windows/main_window.c (hunk 1 above)
Comment (C-D-Lewis) [inline pebble/src/windows/main_window.c:81]: I'm not super concerned about this, since nothing is beneath it. Let me know if you feel strongly.

**Comment by orviwan** [comment]
👍

**Pull request merged**